addons/website/static/src/scss/primary_variables.scss

```diff
@@ -2028,13 +2028,7 @@ $o-font-aliases-to-keys: (
     'navbar': 'navbar-font',
     'buttons': 'buttons-font',
 );
-$o-website-values-palettes: (
-    (
-        'headings-font': 'Source Sans Pro',
-        'navbar-font': 'Source Sans Pro',
-        'buttons-font': 'Source Sans Pro',
-    ),
-) !default;
+$o-website-values-palettes: () !default;
```

Comment:
@stefanorigano Doing this is far from enough to use system fonts (if I am not mistaken, doing this you will just use Roboto (loaded from google) for the whole website as the first font defined in the website map will be used). I have already done a PR to use system fonts for the website here: https://github.com/odoo/odoo/pull/100867 so maybe remove this website change and focus on the backend only :+1: Thanks :+1:

Reply:
Oh... I knew you made some tests but I didn't know you had already a PR 👍 

Should we define `$o-system-font-stack` globally? Or you prefer to keep it under control on website?

Reply:
Mmh, I guess we could indeed. Although, you would still need to do `$o-font-family-sans-serif: o-add-unicode-support-font($o-system-font-stack);` (because the unicode support is automatically added in website fonts later so I cannot put it directly when I defined the variable). Should also rename it `$o-system-fonts` to be consistent with the rest of my PR actually.

Ok: I'll move the variable in web in my PR (RDE will soon merge it, like monday maximum) then you'll be able to rebase your branch afterwards to use the shared variable in the backend, seems ok to you ? :+1:

Reply:
If you plan to stay strict to BS is probably better to define a variable for the frontend actually:
- `Arial` is not good for the backend; semibold is missing and we need it for our 13px buttons that looks too bulky when using **bold**; so I moved it at the end of the list
- We want Ubuntu font to match that OS
-  I've read weird things about the `system-ui`, but maybe were just old news , I should check

Reply:
No, I think it's better to share. If you fix/improve the font system stack for the backend, it probably make sense for the website too. As those are fonts that are supposed to be in the system, I don't see a change you would want to the backend and not the website.

I am changing my PR using the same font stack as bootstrap, once it is merged, you'll be free to modify the font stack as you wish. I am also fixing stuff I missed:
- I forgot to use `o-add-unicode-support-font` on the "theme misconfiguration fallback" I made (thought of it explaining it to you so thanks :p)
- When website is not installed, the frontend use the bootstrap font stack... but without the unicode support of Odoo (I guess this could be a fix in stable, but let's merge it for 16.0 in my branch as a first step, not sure it is worth changing in stable).
- The `o-add-unicode-support-font` actually does not work the right way on the font stack of bootstrap. It puts the unicode support in between emoji fonts while it should be before serif/sans-serif (could also be a fix in stable but there is no broken case in stable since we don't use that font stack).

Reply:
=> https://github.com/odoo/odoo/pull/100867#issuecomment-1256483248

Once RDE has merged that PR, you can thus rebase this one and use the variable I defined in web and you can modify it in whatever way you like :+1:

Reply:
Perfect, thanks 👍